ruby-challenges.rb

```diff
@@ -10,6 +10,20 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+def even_odd num
+    if num % 2 == 0
```

Comment:
Ruby has some cool built in methods that can help with this problem as well.  Check out .even? and .odd?